policyengine_us/parameters/gov/states/wi/tax/income/deduction/phase_out/head_of_household.yaml

```
@@ -0,0 +1,24 @@
+description: Wisconsin deduction is phased-out above this AGI for HEAD_OF_HOUSEHOLD filing units.
```

And similarly below.
```suggestion
description: Wisconsin phases out its standard phased-out at this schedule with respect to AGI for head of household filers.
```

---

Done using a slightly different sentence.